jsk_unitree_robot/jsk_unitree_startup/autostart/imageai.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
Is this a copy from unitree robot? If so please use install.sh to download and copy it.Do we need to install this for Air?

Reply:
Sorry. This change is no longer necessary.
This was to publish the image with ROS.

Instead, I used mqtt to publish.
https://github.com/k-okada/jsk_robot/pull/69/commits/9b2b375d5dab45acad4f4a914dcf37d1d44c2070